codeql/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder:latest
```

Comment:
Is this dockerfile still used/necessary?
I'm afraid next `apt-get` calls will failded because the new image is based on alpine.

Reply:
removed local codeql in: https://github.com/iotaledger/ledger-iota-app/pull/34